### PR TITLE
Feature/review response detail list

### PR DESCRIPTION
**Pull request by amritpurshotam**
Resolves: #864 

Note. Make sure to hide whitespace changes when reviewing. The merge seemed to think the whole api.py file changed.

### CODE REVIEW

File: api/app/reviews/api.py [hunk 1]
```diff
@@ -1,580 +1,650 @@
...
+            question_translation = answer.question.get_translation('en')
+            LOGGER.warn('Could not find {} translation for question id {}'.format(language, answer.question.id))
+        
+        return {
+            'headline': question_translation.headline,
+            'value': answer.value_display
+        }
+    
+    @staticmethod
+    def _serialise_scores(review_score, language):
```
Comment (avishkar58) [inline api/app/reviews/api.py:596]: should be `_serialise_score` (not plural) :)

File: api/app/reviews/repository.py [hunk 2]
```diff
@@ -257,6 +257,15 @@ def delete_response_reviewer(response_id, reviewer_user_id):
         db.session.query(ResponseReviewer).filter_by(response_id=response_id, reviewer_user_id=reviewer_user_id).delete()
         db.session.commit()
 
+    @staticmethod
+    def get_all_review_responses_by_event(event_id):
+        return (
+            db.session.query(ReviewResponse)
+            .join(ReviewForm, ReviewForm.id == ReviewResponse.review_form_id)
+            .join(ApplicationForm, ApplicationForm.id == ReviewForm.application_form_id)
+            .filter_by(event_id=event_id)
```
Comment (avishkar58) [inline api/app/reviews/repository.py:266]: Is the `.all()` not needed? I guess not since the tests are working

**Comment by avishkar58** [comment]
Excellent work as usual, thanks @amritpurshotam ! Great to have you back! :D

**Review by avishkar58 — APPROVED** [review]

File: api/app/reviews/api.py (hunk 1 above)
Comment (amritpurshotam) [reply api/app/reviews/api.py]: Ah yes good catch!. Fixed now.

File: api/app/reviews/repository.py (hunk 2 above)
Comment (amritpurshotam) [reply api/app/reviews/repository.py:266]: Good catch. That is strange how the tests are still passing. I've added the `.all()` now just to be safe.

**Pull request merged**